dynamiclinks/DynamicLinksExampleSwift/AppDelegate.swift

```diff
@@ -68,8 +68,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                    restorationHandler: @escaping ([UIUserActivityRestoring]?) -> Void) -> Bool {
     let handled = DynamicLinks.dynamicLinks().handleUniversalLink(userActivity.webpageURL!) { (dynamiclink, error) in
       // [START_EXCLUDE]
-
-    // [END_EXCLUDE]
+      if((dynamiclink) != nil){
```

Comment:
Remove parentheses and use optional unwrap
```swift
if let dynamiclink = dynamiclink {
  self.handleDynamicLink(dynamiclink)
}
```

Reply:
Done